_docs-sources/reference/services/intro/deploy-new-infrastructure.md

```diff
@@ -0,0 +1,358 @@
+# Deploy new infrastructure
+
+1. [How to deploy Terraform code from the Service Catalog](#how-to-deploy-terraform-code-from-the-service-catalog)
+1. [How to build machine images using Packer templates from the Service Catalog](#how-to-build-machine-images-using-packer-templates-from-the-service-catalog)
+
```

Comment:
```suggestion
```

Proposing to cut these, since we now have the TOC with the same outline.

Reply:
Agreed